app/javascript/controllers/toggle_field_controller.js

```diff
@@ -17,4 +17,16 @@ export default class extends Controller {
       this.checkTarget.parentElement.classList.remove("d-none")
     }
   }
+
```

Comment:
added this in this already existing controller; i wasn't sure it needed to be in a completely different controller, and it shares a little bit of conceptual similarities to what is already here, which is why i put it in here.